src/matchmaking/output/output_fastchess.hpp

```diff
@@ -34,38 +37,57 @@ class Fastchess : public IOutput {
            << "L - "        //
            << stats.draws   //
            << "D ["         //
-           << Elo::getScoreRatio(stats.penta_WW, stats.penta_WD, stats.penta_WL, stats.penta_DD,
-                                 stats.penta_LD, stats.penta_LL)  //
+           << Elo::getScoreRatio(stats.wins, stats.losses, stats.draws)  //
```

Comment:
doesnt this also need to be dynamic?

Reply:
score ratio for pentanomial and wdl are the same when the number of games are even, since when penta_report=true stats are only printed every two games, the wdl function can be used